x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/groq/completion/GroqChatCompletionTaskSettingsTests.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.services.groq.completion;
+
+import org.elasticsearch.common.io.stream.BytesStreamOutput;
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xpack.inference.services.openai.OpenAiServiceFields;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class GroqChatCompletionTaskSettingsTests extends ESTestCase {
```

Comment:
```suggestion
public class GroqChatCompletionTaskSettingsTests extends AbstractWireSerializingTestCase<GroqChatCompletionTaskSettings> {
```

Please make this an `AbstractWireSerializingTestCase` to test the wire serialisation.